modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -198,7 +198,44 @@ controller-0$ sudo docker restart octavia_worker
 
 [NOTE]
 ====
-Depending on your OpenStack environment, Octavia might not support UDP
-listeners, which means there is no support for UDP Services if Kuryr SDN is
-used.
+Depending on your OpenStack environment, Octavia may not support UDP
```

Comment:
"may" - do you mean it is disallowed, or that it's possible? If the latter, then s/may/might.
**may** _v_
Use only to indicate permission. Otherwise, use “can” or “might.”

Reply:
no no, it means it is not supported! It was not implemented at the time and Red Hat only supports it from OSP16